app/css/interceptor.css

```diff
@@ -841,3 +843,243 @@ svg.spinner > circle {
 	border-top-right-radius: 0.25rem;
 	background-color: var(--window-header-bg-color);
 }
+
+
+/* TODO: remove override once bulma dependency is minimized */
```

Comment:
what do you mean by this?

Reply:
bulma adds color to "strong" element (not very semantic) that ironically needs to be reset. We've touched on in some discussion about stripping bulma to a minimum at some point. Created an issue #1021